torch/utils/tensorboard/writer.py

```diff
@@ -246,6 +263,21 @@ def add_scalar(self, tag, scalar_value, global_step=None, walltime=None):
             global_step (int): Global step value to record
             walltime (float): Optional override default walltime (time.time())
               with seconds after epoch of event
+
+        Examples::
+
+            from torch.utils.tensorboard import SummaryWriter
+            writer = SummaryWriter()
+            x = range(100)
+            for i in x:
+                writer.add_scalar('y=2x', i * 2, i)
+            writer.close()
+
+        Expected result:
+
+        .. image:: _static/img/tensorboard/add_scalar.png
+           :scale: 50 %
+
```

Comment:
For add_scalars below can we also add the preample

```
            from torch.utils.tensorboard import SummaryWriter
            writer = SummaryWriter()
```

?

Reply:
also added add_scalars to tensorboard.rst